src/m365/spo/commands/contenttype/contenttype-add.ts

```diff
@@ -75,8 +77,31 @@ class SpoContentTypeAddCommand extends SpoCommand {
           cb(new CommandError(response.ErrorInfo.ErrorMessage));
           return;
         }
-        
-        cb();
+
+        const options: SpoContentTypeGetCommandOptions = {
+          webUrl: args.options.webUrl,
+          listTitle: args.options.listTitle,
+          id: args.options.id,
+          output: args.options.output ?? 'json',
```

Comment:
We can update this to always return JSON from the `spo contenttype get` command.

```suggestion
          output: 'json',
```

Reply:
done